From 6825871e2c904debe5bf137388e6608d1a8415f3 Mon Sep 17 00:00:00 2001
From: Kevin Hilman <khilman@ti.com>
Date: Tue, 4 Jan 2011 15:33:08 -0800
Subject: [PATCH 01/45] MAINTAINERS: change Kevin's email for OMAP PM section

Change my email to TI email address for OMAP PM maintenance.

Signed-off-by: Kevin Hilman <khilman@ti.com>
---
 MAINTAINERS |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 531c5cf..1c56942 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -4475,7 +4475,7 @@ S:	Maintained
 F:	arch/arm/*omap*/*clock*
 
 OMAP POWER MANAGEMENT SUPPORT
-M:	Kevin Hilman <khilman@deeprootsystems.com>
+M:	Kevin Hilman <khilman@ti.com>
 L:	linux-omap@vger.kernel.org
 S:	Maintained
 F:	arch/arm/*omap*/*pm*
-- 
1.7.1

